d2format/format_test.go

```diff
@@ -810,6 +810,21 @@ steps: {
     step-1-content
   }
 }
+`,
+		},
+		{
+			name: "shebang",
+			in: `#!shebang
```

Comment:
this isn't the use case. what's the point of a shebang in a d2 file?

it's that shebangs can exist in code blocks (bash scripts).

your example said that autfmt doesn't format `x ---> y`? https://github.com/terrastruct/d2/issues/1319#issuecomment-1626033015 . that means something is stopping fmt from running in code blocks right. it just needs to extend to `#` too then.

Reply:
Explained the issue in slack